common/amundsen_common/utils/atlas.py

```diff
@@ -247,3 +247,47 @@ def amundsen_key(self) -> str:
             return self._raw_identifier
         else:
             raise ValueError(f'Value is neither qualified name nor amundsen key: {self._raw_identifier}')
+
+
+class AtlasDashboardKey(AtlasKey):
```

Comment:
why do we need this? The assumption for dashboard qualified name was that it's 1:1 with Amudsen key

Reply:
fixed